bazel_tools/haskell-threaded-rts.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/haskell/private/pkgdb_to_bzl.py b/haskell/private/pkgdb_to_bzl.py
+index bb074bb..d8fa971 100755
+--- a/haskell/private/pkgdb_to_bzl.py
++++ b/haskell/private/pkgdb_to_bzl.py
+@@ -55,6 +55,10 @@ def hs_library_pattern(name, mode = "static", config = ""):
+     """
+     if name == "Cffi" and mode == "dynamic":
+         libname = "ffi"
++    elif name == "HSrts":
```

Comment:
Why do we need this patch? Judging from the comments on this function, it seems like it is already supposed to handle `Hsrts` in some way. Why do we need to handle it differently? Surely we are not the only once using the threaded RTS.